dashboard/app/helpers/ai_diff_bedrock_helper.rb

```diff
@@ -97,6 +105,10 @@ def self.filter_for_context(config, lesson_number, unit_num, course_name)
     end
     temp_filters.push({equals: {key: "course", value: course_name}}) unless course_name.nil?
 
+    if lesson_number.nil? && unit_num.nil? && course_name.nil?
+      temp_filters.push({equals: {key: "scope", value: "general"}})
+    end
+
```

Comment:
I know we talked about adding `GENERAL` to course/unit/lesson as well, and I think that still makes sense.  I wanted to get this PR out though first for Tess to play with the "only general" context.

Reply:
Update: [This](https://codedotorg.slack.com/archives/C050HE4QDHR/p1740688232956829?thread_ts=1740687973.334979&cid=C050HE4QDHR) thread from Tess says to hold off on using the GENERAL docs with the other docs...